src/domUpdates.js

```diff
@@ -253,6 +290,24 @@ const createIngredientsHTML = ingredients => {
     </label>
     `;
   });
+};
+
+const switchView = (clickedViewID) => {
+  if (clickedViewID === "our-recipes") {
+    // allUserRecipes.classList.add("hidden");
+    recipeGrid.classList.remove("hidden");
+    ourViewBtn.classList.add("selected-view");
+    yourViewBtn.classList.remove("selected-view");
+    yourViewBtn.classList.add("unselected-view");  
+    ourViewBtn.classList.remove("unselected-view");
+  } else {
+    // allUserRecipes.classList.remove("hidden");
```

Comment:
What's happening with this commented out code? Do we still need this? Also, allUserRecipes is imported at the top of this file but never used outside of comments.

Reply:
Hey i built this before our data model discussion so imagined using allUserRecipes as a separate view. I'll remove it.